README.md

```diff
@@ -1169,7 +1169,7 @@ Defaults:
   errorDataPath:    'object', // deprecated
   messages:         true,
   sourceCode:       false,
-  processCode:      undefined, // function (str: string): string {}
+  processCode:      undefined, // function (str: string, schema: object): string {}
```

Comment:
Please also add comment in the lines 1281-2 below. `require('js-beautify').js_beautify` probably won't work anymore after this change, as it probably has some optional second parameter (it needs to be checked), so it has to be mentioned both in the docs and in the release notes what it has to be replaced with - not sure how many users would be affected... I am thinking to release over the next weekend to see if anything breaks. Alternatively it can be released on the `beta` branch in preparation for the next major version. I will think about it, but please add to the docs in any case.